docs/pages/machine-id/guides/databases.mdx

```diff
@@ -0,0 +1,284 @@
+---
+title: Machine ID with Databases
+description: Protecting your databases with Machine ID
+---
+
+In this guide, we will demonstrate how to use Machine ID to access a database
+protected by Teleport from a custom application.
+
+You may already be using Teleport Database Access to protect your database
+when humans access it. Machine ID can be used to protect your data when
+machines, like web applications, access your data as well. With Machine ID you
+will be able to get the same short-lived certificates tied to a machine
+identity that can be rotated, audited, and controlled with all the familiar
+Teleport access controls.
+
+<Figure align="left" bordered caption="Machine ID and Database Access Deployment">
+  ![Machine ID and Database Access Deployment](../../../img/machine-id/machine-id-database-app.png)
+</Figure>
+
+## Prerequisites
+
+You will need Teleport  >= 9.3.0 
```

Comment:
```suggestion
You will need Teleport >= 9.3.0 
```

Reply:
I've switched this over to a `ScopedBlock`